resources/benchmark-report.mjs

```diff
@@ -1,83 +1,107 @@
 // This file can be customized to report results as needed.
 
 (function () {
-    if (!window.testRunner && location.search != '?webkit' && location.hash != '#webkit')
+    if (
+        !window.testRunner &&
+        location.search !== "?webkit" &&
+        location.hash !== "#webkit"
+    )
         return;
 
-    if (window.testRunner)
-        testRunner.waitUntilDone();
+    if (window.testRunner) window.testRunner.waitUntilDone();
```

Comment:
This is not right either. Each statement should be on its own line:
https://webkit.org/code-style-guidelines/#linebreaking-multiple-statements